enterprise/backend/src/metabase_enterprise/advanced_config/util/urls.clj

```diff
@@ -0,0 +1,14 @@
+(ns metabase-enterprise.advanced-config.util.urls
+  (:require [metabase.models.setting :as setting :refer [defsetting]]
+            [metabase.public-settings :as public-settings]
+            [metabase.public-settings.premium-features :as premium-features]
+            [metabase.util.i18n :refer [deferred-tru]]))
+
+(defsetting pulse-url
+  (deferred-tru "By default \"Site Url\" is used in notification links, but can be overridden."))
```

Comment:
`defsetting` accepts a `:getter` function, which I think would be a better place to put the logic in `site-url` rather than having a separate wrapper function. There are a number of similar examples in the `metabase.public-settings` namespace.